packages/cozy-doctypes/src/Document.js

```diff
@@ -75,7 +81,29 @@ class Document {
     }
   }
 
+  static unregisterClient() {
+    this.cozyClient = null
+  }
+
+  /**
+   * Returns true if Document uses a CozyClient (from cozy-client package)
+   *
+   * @returns {boolean} true if Document uses a CozyClient
+   **/
+  static usesCozyClient() {
+    return this.cozyClient instanceof CozyClient
+  }
+
   static getIndex(doctype, fields) {
+    if (this.usesCozyClient()) {
+      throw new Error('This method is not implemented yet with CozyClient')
+    }
+
+    return this._getIndex(doctype, fields)
+  }
+
+  // getIndex method that uses cozy-client-js
```

Comment:
getIndexViaOldClient lets you remove the comment.

Reply:
ok for the renaming. As I said in the description, I wanted to prefix those methods with `_ccjs_` but I did not like it. `*ViaOldClient` is more clear